contracts/asset-proxy/contracts/src/libs/LibAssetData.sol

```diff
@@ -0,0 +1,228 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+import "@0x/contracts-erc1155/contracts/src/interfaces/IERC1155Mintable.sol";
```

Comment:
Ignorance. :)  I guess because I wrote the tests using a `Mintable` I then wrote the implementation using the same interface.  TDD for the lose! :)  Fixed in https://github.com/0xProject/0x-monorepo/pull/1779/commits/f2055d215da9670e0d5a061e7cbaeee22ebddd5d .